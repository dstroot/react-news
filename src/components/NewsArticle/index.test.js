import React from "react";
import renderer from "react-test-renderer";

import NewsArticle from "../NewsArticle";
const story = `## Heading

Here is some content

- List 1
- List 2`;

describe("NewsArticle", () => {
  it("it should render", () => {
    const component = renderer.create(
      <NewsArticle
        date="Dec 12, 2018"
        cutline="This is a cutline"
        headline="This is a headline!"
        story={story}
      />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
