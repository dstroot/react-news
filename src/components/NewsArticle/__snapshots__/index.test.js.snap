// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsArticle it should render 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <h5
        className="text-uppercase text-primary mb-0 mt-5"
      >
        Press Release
      </h5>
      
      <p>
        Dec 12, 2018
      </p>
      <h1
        className="display-6 mt-5"
      >
        This is a headline!
      </h1>
      <p
        className="lead mt-4"
      >
        This is a cutline
      </p>
      <hr
        className="m-4"
      />
      <h2>
        Heading
      </h2>
      <p>
        Here is some content
      </p>
      <ul>
        <li>
          List 1
        </li>
        <li>
          List 2
        </li>
      </ul>
      <hr
        className="m-4"
      />
      <div
        className="row"
      >
        <div
          className="col"
        >
          <h3>
            Press Contacts
          </h3>
          <address
            className=""
          >
            <strong>
              Media Contact
            </strong>
            <br />
            <a
              className="card-link"
              href="mailto:media@pacificlife.com?Subject=Media%20Inquiry"
              target="_top"
            >
              media@pacificlife.com
            </a>
            <br />
            <a
              className="card-link"
              href="tel:+19492194575"
            >
              (949) 219-4575
            </a>
          </address>
        </div>
        <div
          className="col"
        >
          <img
            alt="logo"
            height="115px"
            src="logo.png"
            width="175px"
          />
        </div>
      </div>
      <hr
        className="m-4"
      />
      <a
        aria-current="page"
        className="btn btn-large btn-secondary mb-5"
        href="/"
        onClick={[Function]}
      >
        Back to Articles
      </a>
    </div>
  </div>
</div>
`;
